app/controllers/tasks/create-task.js

```diff
@@ -1,43 +1,79 @@
 import Controller from '@ember/controller';
 import { service } from '@ember/service';
 import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
 
 export default class TasksCreateTaskController extends Controller {
   @service router;
   @service store;
   @service('requests/task/task-service') requestTaskService;
 
+  @tracked selectedPriority = 'low';
+  @tracked selectedStatus = 'new';
+
+  get priorityOptions() {
+    return ['low', 'normal', 'high', 'urgent'];
```

Comment:
You create the same getter twice (in controllers) and then pass it to `<TaskOptions/>` component - imo it is redundant.
If you already created a specific component :clap:  `<TaskOptions/>` it can keep these values, so we will have only one place with this hard codded array of values.
So you have 2  options:
* create 2 new components which will e.g.:
`<TaskPriorityOptions/>` <--- inside component, you keep priorities and use `<TaskOptions/>` component
`<TaskStatusOptions/>` <--- inside component, you keep status values and use `<TaskOptions/>` as well.

* create `@variation` parameter to `<TaskOptions/>` component, and keep priority and status values inside, and depending on the `@variation` use proper values.

Reply:
I have changed it to house the status and priority in the component for now, but will come back to this and have another go at adding an options element and probably separate components for this - so essentially I am using your second solution above:

RE: https://github.com/Mr-Smyth/task-manager/pull/25/commits/ea96aebcff3e33e7280dbb2ecc0cd389aa0a6c00

Reply:
Added new components for these 2 (priority and status) as selection boxes and removed a lot of unnecessary code id added when trying to get my previous approach to work:

RE: https://github.com/Mr-Smyth/task-manager/pull/25/commits/c7e666015ea251c3f1983975dbd8741675a9298e